be/src/exec/chunks_sorter_topn.cpp

```diff
@@ -124,6 +124,10 @@ std::vector<JoinRuntimeFilter*>* ChunksSorterTopn::runtime_filters(ObjectPool* p
     // _topn_type != TTopNType::RANK means we need reserve the max_value
     bool is_close_interval = _topn_type == TTopNType::RANK || _sort_desc.num_columns() != 1;
 
+    if (order_by_column->is_null(current_max_value_row_id)) {
```

Comment:
why need this check now?

Reply:
after https://github.com/StarRocks/starrocks/pull/30591 we have support nulls last case. so the order by column may be a nullable-column. we should check heap top shouldn't be a null vallue